org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeSerialPort.java

```diff
@@ -179,7 +189,6 @@ public void close() {
         try {
             if (serialPort != null) {
                 serialPort.removeEventListener();
-                serialPort.enableReceiveTimeout(1);
```

Comment:
I have removed it because of this exception:

```
2018-12-14 09:33:54.755 [DEBUG] [.z.a.d.ZigBeeNetworkDiscoverer:129  ] - Network discovery task: shutdown
2018-12-14 09:33:54.755 [DEBUG] [z.a.d.ZigBeeDiscoveryExtension:105  ] - DISCOVERY Extension: Shutdown
2018-12-14 09:33:54.758 [ERROR] [o.b.z.handler.ZigBeeSerialPort:212  ] - Error closing serial port: '/dev/ttyUSB0' java.io.IOException: Input/output error in TimeoutThreshold
        at gnu.io.RXTXPort.NativeEnableReceiveTimeoutThreshold(Native Method)
        at gnu.io.RXTXPort.enableReceiveTimeout(RXTXPort.java:406)
        at org.openhab.binding.zigbee.handler.ZigBeeSerialPort.close(ZigBeeSerialPort.java:192)
        at com.zsmartsystems.zigbee.dongle.ember.ZigBeeDongleEzsp.shutdown(ZigBeeDongleEzsp.java:443)
        at com.zsmartsystems.zigbee.ZigBeeNetworkManager.shutdown(ZigBeeNetworkManager.java:538)
        at org.openhab.binding.zigbee.handler.ZigBeeCoordinatorHandler$1.run(ZigBeeCoordinatorHandler.java:505)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)

2018-12-14 09:33:54.759 [DEBUG] [.z.z.d.e.i.ash.AshFrameHandler:372  ] - AshFrameHandler close.
```

It also occurs if the threads are synchronized. I am wondering if this Exception also occurred before my changes and just popped to the surface as I now shutdown and reconnect periodically.

Reply:
So I guess the question is - have you removed something that was required, to stop the exception, and we now have created another problem, or was this method just not needed in the first place?  Personally, I'm not sure without thinking about it some more and checking what it really does, but I think this needs to be checked.

Reply:
I think that setting it to 1 was done to prevent delays/lockups during the close. I'm just surprised that you are now getting an exception with this when it never has been a problem before. Any idea why this is?
There has been a lot of work to get the serial port working, and I'm just worried that changing this sort of thing without understanding the issue will lead to further problems. Do you understand what has changed that now causes this exception?